_targets.R

```diff
@@ -0,0 +1,45 @@
+library(targets)
+source('./1_fetch/src/getData.R')
```

Comment:
Since you have this set up in an RStudio project, you shouldn't need the `'./'` in your filepath. It should work to just write `source('1_fetch/src/getData.R')`

Reply:
This applies to all of your filepaths in this project